buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -25,7 +25,7 @@ base {
 
 java {
   toolchain {
-    languageVersion.set(JavaLanguageVersion.of(11))
+    languageVersion.set(JavaLanguageVersion.of(17))
```

Comment:
This would require a README/CONTRIBUTING.md update since it adds a new requirement for the build-environment. Or would gradle auto-download Java 17?

Reply:
Gradle auto downloads indeed

Reply:
So i.e., it's still just min Java 11 to run the build